package.json

```diff
@@ -425,6 +425,7 @@
     "nock": "10.0.6",
     "node-sass": "^4.9.4",
     "normalize-path": "^3.0.0",
+    "null-loader": "^3.0.0",
```

Comment:
**operations**: added a null loader to the runtime webpack config to nullify some irrelevant imports that were bloating the runtime.